api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.4.x.xml

```diff
@@ -86,6 +86,11 @@
         <sql> CREATE EXTENSION IF NOT EXISTS fuzzystrmatch SCHEMA public;</sql>
     </changeSet>
     
+    <changeSet id="20200715-uuid_ossp_extension" author="aman" dbms="postgresql">
+        <comment> Extension to use UUID functions with PostgreSQL</comment>
+        <sql> CREATE EXTENSION IF NOT EXISTS "uuid-ossp" SCHEMA public;</sql>
```

Comment:
Do we need the double quotes?

Reply:
@dkayiwa , Yes, else you will get an error due to "-" in uuid-ossp .